src/test/unit_tests/test_nc_health.js

```diff
@@ -158,13 +173,20 @@ mocha.describe('nsfs nc health', function() {
 
         mocha.afterEach(async () => {
             await fs_utils.file_delete(config_fs.config_json_path);
+            restore_health_if_needed(Health);
         });
 
-        mocha.it('Health all condition is success', async function() {
-            const get_service_state = sinon.stub(Health, "get_service_state");
+        mocha.it('Health all condition is success1', async function() {
```

Comment:
Was this "1" intentional? 
```suggestion
        mocha.it('Health all condition is success', async function() {
```